src/Evolution/Systems/GrMhd/ValenciaDivClean/FiniteDifference/PositivityPreservingAdaptiveOrder.hpp

```diff
@@ -164,38 +164,44 @@ class PositivityPreservingAdaptiveOrderPrim : public Reconstructor {
                          const PositivityPreservingAdaptiveOrderPrim& rhs);
 
   double four_to_the_alpha_5_ = std::numeric_limits<double>::signaling_NaN();
+  double six_to_the_alpha_7_ = std::numeric_limits<double>::signaling_NaN();
```

Comment:
Just to be clear, are you saying I should squash the last two commits together? I kept them separate since one is generic FD and the other is a specific system, but I don't really care :)